.github/actions/restore-hermes-workspace/action.yml

```diff
@@ -1,8 +1,13 @@
-name: setup_hermes_workspace
-description: "Setup hermes workspace"
+name: restore-hermes-workspace
```

Comment:
cherry picked 9c6c63781669bd1d6451c880671a151060220020 for stabilize prepare-hermes-workspace